src/backends/torch/torchsolver.cc

```diff
@@ -277,6 +277,25 @@ namespace dd
     // reload solver if asked for and set it value accordingly
     if (ad_mllib.has("resume") && ad_mllib.get("resume").get<bool>())
       {
+        if (mlmodel._sstate.empty())
+          {
+            std::string msg
+                = "resuming a model requires a solverstate file in model "
```

Comment:
`solverstate` was the file extension for caffe, I suggest we change it here so it is relevant for torch.

Reply:
ok, done